src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -134,10 +107,25 @@ public void onPreInit(FMLPreInitializationEvent e) {
     @Subscribe
     public void onInitialization(FMLInitializationEvent e) {
         this.registry.init();
+        try {
+            getGame().getServiceManager().setProvider(this, CommandService.class, new SimpleCommandService(getGame().getPluginManager()));
```

Comment:
https://github.com/SpongePowered/Sponge/pull/144/files#diff-58e0014cf676b44128730ca938af8f0dR51
and 
https://github.com/SpongePowered/Sponge/pull/144/files#diff-877cd7506b19b24f60c4cdfd0180cf3cR60